app/Models/Post.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Post extends Model
+{
+    use HasFactory;
+    protected $fillable = ['name', 'description', 'user_id'];
```

Comment:
Because you used 
`$post = new Post();`
Rather than:
`$post = Post::create()`
You don't need to set fillable properties.